src/frisby/spec.js

```diff
@@ -110,14 +110,14 @@ class FrisbySpec {
   }
 
   _fetchParams(params = {}) {
-    let fetchParams = _.merge({}, this._setupDefaults.request, params);
+    let fetchParams = _.cloneDeep(this._setupDefaults.request);
 
     // Form handling - send correct form headers
     if (params.body instanceof FormData) {
-      fetchParams.headers = _.merge(fetchParams.headers, fetchParams.body.getHeaders());
+      delete fetchParams.headers['Content-Type'];
```

Comment:
Doesn't this effectively do the same thing as the code that is already there?

Reply:
Thank you for your review.

`Content-Type` is supported in libraries.
- [node-fetch](https://github.com/bitinn/node-fetch#post-with-form-data-detect-multipart)
- [form-data](https://github.com/form-data/form-data#node-fetch)

And it looks like getHeaders() is non-standard API.